snapcraft/internal/repo/_deb.py

```diff
@@ -289,7 +287,16 @@ def get_package_libraries(cls, package_name: str) -> Set[str]:
 
     @classmethod
     def get_package_for_file(cls, file_path: str) -> str:
-        return _run_dpkg_query_search(file_path)
+        try:
+            absolute_file_path = os.path.join(os.path.sep, file_path)
```

Comment:
absolute_file_path would be nice if it were a pathlib.Path type
```suggestion
            absolute_file_path = filepath.Path("/") / file_path
```

The might be a better way of doing `filepath.Path("/")`